frontend/src/components/QueryBuilder/Search/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { SearchOutlined } from '@ant-design/icons';
+import { Button, Select, Space } from 'antd';
+import { QUERY_BUILDER_STATE_KEYS } from 'constants/queryBuilder';
+import { useQueryBuilderContext } from 'container/QueryBuilder';
+import { useAutoComplete } from 'hooks/queryBuilder/useAutoComplete';
+import React, { useEffect } from 'react';
+
+import { Tag } from './tag';
+
+function QueryBuilderSearch(): JSX.Element {
+	const { onChangeHandler, onSubmitHandler } = useQueryBuilderContext();
+	const { Option } = Select;
```

Comment:
donot define here. as rerendering would be re-calculating this object... so use Select.Option directly